packages/browser/src/core/analytics/index.ts

```diff
@@ -115,6 +116,13 @@ export class Analytics extends Emitter {
     this.integrations = options?.integrations ?? {}
     this.options = options ?? {}
 
+    inspectorHost.start({
```

Comment:
Can this be imported and loaded lazily (e.g. on devtools open / connection) in order that it doesn't affect bundle size?

Reply:
@zikaari Maybe not worth investigating, I think the stakes are super low.

The way the module is being injected -- means that it actually won't be loaded unless you have the devtools extension installed (basically, you're a developer =)

Nice job.

Reply:
Not sure it's worth doing. I think to make this a plugin, you'd need to make it the `utility` type and make the `load` method start the host and attach event listeners for the API calls (track, identify, page, group, alias, screen), then call `inspectorHost.trace` with the context.

But I don't fully understand why `inspectorHost.trace` is being called in-between plugin lifecycles and/or if the context has enough data so those calls can be removed and triggered once when the event is delivered instead.

Reply:
There might be a benefit to allowing the `inspectorHost` to be passed to the `Analytics` constructor. Right now it only works in browsers - and we're starting to break up our repo so that the `core` isn't specific to a single runtime.

Reply:
Yep. Much of “core” is going to be moved into its own package, and what will remain will be browser-specific. Goal is to remove anything that references the window / document object from core.